### PR TITLE
Update schema for import

### PR DESCRIPTION
**Pull request by Jougan-0**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: schemas/configuration/generate.json [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+                    "Machine Learning",
+                    "Observability and Analysis",
+                    "Orchestration & Management",
+                    "Platform",
+                    "Provisioning",
+                    "Runtime",
+                    "Security & Compliance",
+                    "Serverless",
+                    "Tools",
+                    "Uncategorized"
```
Comment (MUzairS15) [inline schemas/configuration/generate.json:104]: These should be fetched dynamically, look for the IMport design model (source type field) for reference on dynamically fetching.

File: schemas/configuration/generate.json [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+                  ]
+                },
+                "subCategory": {
+                  "type": "string",
+                  "enum": [
```
Comment (MUzairS15) [inline schemas/configuration/generate.json:109]: same here

**Review by MUzairS15 — CHANGES_REQUESTED** [review]

File: schemas/configuration/generate.json (hunk 1 above)
Comment (Jougan-0) [reply schemas/configuration/generate.json:104]: @MUzairS15 We used enum here to :thinking: 
https://github.com/meshery/meshkit/blob/fe3024a1caa43e6e9b281c326dac83bb93f93df7/schemas/configuration/designImport.json#L13

Comment (MUzairS15) [reply schemas/configuration/generate.json:104]: Then it must be some other schema, check catalog publish

Comment (Jougan-0) [reply schemas/configuration/generate.json:104]: I'll make them dynamically after today's event.

**Pull request merged**